tools/core-build-tasks/package.json

```diff
@@ -19,6 +19,7 @@
   "scripts": {
     "build-tools": "just-scripts build-tools",
     "clean-tools": "just-scripts clean-tools",
+    "lint": "just lint",
```

Comment:
nit: `just-scripts lint`

Just a tad bit more portable for some platforms or install set ups

Reply:
Fixed, thanks